src/transformers/pipelines/automatic_speech_recognition.py

```diff
@@ -250,9 +285,11 @@ def preprocess(self, inputs, chunk_length_s=0, stride_length_s=None):
             raise ValueError("We expect a single channel audio input for AutomaticSpeechRecognitionPipeline")
 
         if chunk_length_s:
-            if self.type not in {"ctc", "ctc_with_lm"}:
-                raise ValueError(
-                    "`chunk_length_s` is only valid for CTC models, use other chunking options for other models"
+            if self.type == "seq2seq":
+                logger.warning(
+                    "Using `chunk_length_s` is very experimental. The results will not necessarily be entirely"
+                    " accurate and will have caveats. More information:"
+                    " https://github.com/huggingface/transformers/pull/20104"
                 )
```

Comment:
Is there already a created way to do that ?

Otherwise I can create some tool for it.
Any other location we could add this "single" warning ? (Will add in a different PR)

Reply:
We use a dict in the state like [this one](https://github.com/huggingface/transformers/blob/07d8d6e2f7a920d399e5e86a82d78179cdfa6746/src/transformers/tokenization_utils_base.py#L1499). No need to overengineer another solution IMO.

Reply:
Done.